components/tikv_util/src/sequence_number.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Why put it in util?

Reply:
Other files also have this header. 🤔

Reply:
What I mean is why put `sequence_number` module in tikv_util? It seems rocksdb related.

Reply:
Moved to engine_traits/util